tasks/main.yml

```diff
@@ -22,8 +22,7 @@
 
 - name: Generate /etc/SUBSYSTEM.conf
   template:
-    src: SUBSYSTEM.conf.j2
-    dest: /etc/SUBSYSTEM.conf
-    backup: yes
+      src: SUBSYSTEM.conf.j2
+      dest: /etc/SUBSYSTEM.conf
+      backup: yes
```

Comment:
Aren't we commonly using a 2-space indentation? Shouldn't we change the rest of the template to 2 spaces, instead of changing this occurrence to 4 spaces?

Reply:
Correct, fixed.